.circleci/config.yml

```diff
@@ -122,21 +111,19 @@ jobs:
           <<: *destroy_device
           when: on_fail
 
-  # TODO: In a future PR, deploy artifact to S3.
   deploy-ova:
     <<: *job_defaults
-    docker:
-      - image: st2opsadmin/ovabuild-circle
     steps:
       - checkout
       - attach_workspace:
           at: ~/ova
+      - run:
+          name: Deploy to GitHub releases
+          command: ghr -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} ${CIRCLE_TAG} builds
```

Comment:
From the https://github.com/StackStorm/packer-st2/releases page, the only missing thing, - we need to upload `.ova` file only, excluding README and `.box`.